src/test/java/org/opentripplanner/street/search/state/TestStateBuilder.java

```diff
@@ -58,6 +73,52 @@ public TestStateBuilder streetEdge() {
     return this;
   }
 
+  /**
+   * Traverse a street edge and switch to Car mode
+   */
+  public TestStateBuilder pickUpCar() {
+    count++;
+
+    var station = new VehicleRentalStation();
```

Comment:
I've copied this code from https://github.com/opentripplanner/OpenTripPlanner/blob/4dd671a1192c03f29eccb626f210d9f4db9c9c88/src/test/java/org/opentripplanner/street/model/edge/VehicleRentalEdgeTest.java#L158-L181 but I think we should create a reusable method to create the instances.